sycl/plugins/unified_runtime/ur/adapters/level_zero/command_buffer.cpp

```diff
@@ -374,6 +374,40 @@ static ur_result_t enqueueCommandBufferMemCopyRectHelper(
   return UR_RESULT_SUCCESS;
 }
 
+// Helper function for enqueuing memory fills
+static ur_result_t enqueueCommandBufferFillHelper(
```

Comment:
You are not checking that the pattern size matches the capability of the copy queues as it is done in regular enqueuing function (memory.cpp l 187 to 201). Is there a reason for this? if not, it may be safer to add these checks using the Device registered in the CommandBuffer.

Reply:
We're not using copy engines for the main command list right now, but I'll add the check for the compute engine capabilities.